tinyiiod.c

```diff
@@ -46,12 +43,12 @@ void tinyiiod_destroy(struct tinyiiod *iiod)
 	free(iiod);
 }
 
-int tinyiiod_read_command(struct tinyiiod *iiod)
+int32_t tinyiiod_read_command(struct tinyiiod *iiod)
 {
 	char buf[128];
-	int ret;
+	int32_t ret;
 
-	ret = (int) tinyiiod_read_line(iiod, buf, sizeof(buf));
+	ret = tinyiiod_read_line(iiod, buf, sizeof(buf));
```

Comment:
Currently, this function returns `uint32_t`, so `ret `will always be greater than 0